apps/gameserver/include/Spawn.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include "Monster.h"
+
+#include <bango/processor/db.h>
+#include <bango/utils/time.h>
+
+//TODO:
+//Monster might not exist in InitMonster -> spawn shouldn't be created
+//Some monsters do not spawn with error cannot insert into quad with error: not in boundary
+
+struct GenMonster : public bango::processor::db_object<GenMonster>
+{
+    struct RectXY
+    {
+        int X1 = 0, Y1 = 0, X2 = 0, Y2 = 0;
+        int GetRandomX() const;
+        int GetRandomY() const;
+
+    };
+    std::uint32_t MonsterIndex = 0, Map = 0, Index = 0, Amount = 0, SpawnCycle = 0;
```

Comment:
```suggestion
    std::uint32_t MonsterIndex = 0, Map = 0, Area= 0, Amount = 0, SpawnCycle = 0;
```
Why not call it as it is inside config file?

Reply:
done